source/module_esolver/esolver_ks_lcao.cpp

```diff
@@ -1242,8 +1259,10 @@ void ESolver_KS_LCAO<TK, TR>::after_scf(UnitCell& ucell, const int istep)
         std::cout << FmtCore::format(" >> Finish %s.\n * * * * * *\n", "Wave function to Wannier90");
     }
 
-    // add by jingan
-    if (PARAM.inp.calculation == "nscf" && berryphase::berry_phase_flag && ModuleSymmetry::Symmetry::symm_flag != 1)
+    //! 20) berry phase calculations, added by jingan
```

Comment:
Note that this PR deletes some developers' names, the following code change can make this PR consistent with itself
```suggestion
    //! 20) berry phase calculations
```